homeassistant/components/squeezebox/media_player.py

```diff
@@ -65,79 +67,118 @@
     | SUPPORT_CLEAR_PLAYLIST
 )
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-        vol.Optional(CONF_USERNAME): cv.string,
-    }
-)
-
-DATA_SQUEEZEBOX = "squeezebox"
-
-KNOWN_SERVERS = "squeezebox_known_servers"
-
+KNOWN_PLAYERS = "known_players"
+KNOWN_SERVERS = "known_servers"
+DISCOVERY_TASK = "discovery_task"
 ATTR_PARAMETERS = "parameters"
-
 ATTR_OTHER_PLAYER = "other_player"
 
 ATTR_TO_PROPERTY = [
     ATTR_QUERY_RESULT,
     ATTR_SYNC_GROUP,
 ]
 
+SQUEEZEBOX_MODE = {
+    "pause": STATE_PAUSED,
+    "play": STATE_PLAYING,
+    "stop": STATE_IDLE,
+}
+
+
+async def start_server_discovery(hass):
```

Comment:
It looks like this function is only called in `__init__.py` so it might be better off there.

Reply:
Ok, moved it.